src/App.jsx

```diff
@@ -14,57 +15,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button data-cy="AddButton" type="button" className="button">
-              +
-            </button>
-          </td>
+  const title =
+  selectedGood ? `${selectedGood} is selected` : `No goods selected`;
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {title}
+        {selectedGood ? (
+          <button
+          data-cy="ClearButton"
+          type='button'
+          className="delete ml-3"
+          onClick={() => setSelectedGood('')}
```

Comment:
```suggestion
          onClick={reset}
```
Consider leveraging DRY principle